apps/emqx_management/src/emqx_mgmt_api_configs.erl

```diff
@@ -66,23 +91,44 @@ schema("/configs") ->
         'operationId' => configs,
         get => #{
             tags => ?TAGS,
-            description => ?DESC(get_conf_node),
+            description => ?DESC(get_configs),
             parameters => [
-                {node,
```

Comment:
origin GET `/configs` is unused, So I think incompatibility here is accepted.

Reply:
I suggest that it should be kept, even if no one is using it?

Then, we can add a check on the HTTP Header of the request. If the `Accept` header is set to `application/json` or is not specified, we will return the response according to the original logic. If the `Accept` header is set to `text/plain`, we will return the response in hocon file format.

Reply:
Okay, I'll change it back right away, although personally I really dislike doing so.